src/utils/spp.ts

```diff
@@ -0,0 +1,48 @@
+import request, { CliRequestOptions } from '../request.js';
+
+export interface ContentCenterSite {
+  WebTemplateConfiguration: string;
+}
+
+export interface SppModel {
+  AIBuilderHybridModelType: string;
+  AzureCognitivePrebuiltModelName: string;
+  BaseContentTypeName: string;
+  ConfidenceScore: string;
+  ContentTypeGroup: string;
+  ContentTypeId: string;
+  ContentTypeName: string;
+  Created: string;
+  CreatedBy: string;
+  DriveId: string;
+  Explanations: string;
+  ID: number,
+  LastTrained: string;
+  ListID: string;
+  ModelSettings: string;
+  ModelType: number,
+  Modified: string;
+  ModifiedBy: string;
+  ObjectId: string;
+  PublicationType: number;
+  Schemas: string;
+  SourceSiteUrl: string;
+  SourceUrl: string;
+  SourceWebServerRelativeUrl: string;
+  UniqueId: string;
+}
+
+export const spp = {
+  async isContentCenter(siteUrl: string): Promise<boolean> {
```

Comment:
Let's add a JSDoc comment to make clear what this function does.

Reply:
Done